README.md

```diff
@@ -36,14 +38,69 @@ var Button = component({
 })
 
 // register as custom element
-Button = toCustomElement(customButton, 'button')
-document.registerElement('custom-button', Button)
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title'])
+document.registerElement('custom-button', CustomButton)
 
 // create new custom-button
-var button = document.createElement('button', 'custom-button')
+var button = document.createElement('custom-button')
 document.body.appendChild(button)
 ```
 
+## Custom Elements (webcomponents-v1)
+```js
+var toCustomElementV1 = require('nanocomponent-adapters/custom-element-v1')
+var component = require('nanocomponent')
+var html = require('bel')
+
+// create new nanocomponent
+var Button = component({
+  render: function (data) {
+    return html`
+      <button>hello planet</button>
+    `
+  }
+})
+
+// register as custom element
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title']);
+customElements.define('custom-button', CustomButton);
+// create new custom-button
+var button = document.createElement('custom-button')
+document.body.appendChild(button)
+```
+
+## Angular
+```js
```

Comment:
For sure, understood (and also enraged by the decision to only use classes for custom elements) 

But (and @yoshuawuyts can correct me here if I'm wrong) - my understanding of nanocomponent is that 
it's a generic interface to create components, in the same way

In the custom-elements v1 I  can see that approach laid out plainly, here I can't see how it relates to using nanocomponent

(on a side note it looks like there are ways to hack around the class restriction - https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js#L98-L153)